config/settings.yml

```diff
@@ -7,3 +7,7 @@ apps_with_migrated_tagging:
  - policy-publisher
  - hmrc-manuals-api
  - travel-advice-publisher
+
+# deprecated apps
+apps_without_migrated_tagging:
```

Comment:
This can also be read as "not yet migrated". Perhaps `apps_without_tagging` is better?